src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/GetAllocationResourceIntTest.kt

```diff
@@ -214,10 +214,12 @@ class GetAllocationResourceIntTest : IntegrationTestBase() {
     }
 
     /*
-     * Note that each test in this class also checks the endpoint `/activities/ids` as well as the allocation endpoint.
-     * This is to test that the same rules are being applied to both the activities and allocations selection and
-     * effectively tests that the custom queries in CourseActivityRepository.findActiveActivities and
-     * OffenderProgramProfilesRepository.findActiveAllocations are aligned.
+     * Note that each test in this class also checks the endpoint `/activities/ids` and `/allocations/suspended` as well
+     * as the allocation endpoint.
+     *
+     * This is to test that the same rules are being applied to the activities, suspended activities and allocations
+     * selection and effectively tests that the custom queries in CourseActivityRepository.findActiveActivities,
+     * OffenderProgramProfilesRepository.findActiveAllocations / findSuspendedAllocations are aligned.
```

Comment:
It's not ideal to be testing 3 endpoints in the same test suite but it's important that the 3 different SQL statements are aligned - if in future somebody changes one of the SQLs then the tests for the other 2 endpoints need to fail as a reminder to update the other SQLs.